### ISSUE TITLE
User manual won't display

### ISSUE THREAD

**Issue by 2011** [MuntashirAkon/AppManager#1088]
### Please check before submitting an issue

- [X] I know what my device, OS and App Manager versions are
- [X] I know how to take logs
- [ ] I know how to reproduce the issue which may not be specific to my device

### Describe the bug

Only have this problem on one of my devices (an Android 7 tablet) - the only device I have with the latest webview (114) installed.

### To Reproduce

Select User Manual from the menu. The screenshot below shows the message that I get.

### Screenshots

![webpage_not_available](https://github.com/MuntashirAkon/AppManager/assets/550150/28acdf0f-a30a-4cfd-832f-4b5f480201fb)

### Logs

Probably not relevant, but I see this warning 202 times in the log:

```
[06-07 20:11:01.342 14651:14710 W/JavaBinder]
BinderProxy is being destroyed but the application did not call unlinkToDeath to unlink all of its death recipients beforehand.  Releasing leaked death recipient: io.github.muntashirakon.io.FileSystemService$$ExternalSyntheticLambda1
```

The log entry generated when I try to open the User Manual:

```
[06-07 20:11:02.924 14536:28997 E/AndroidProtocolHandler]
Unable to open resource URL: file:///android_res/raw/index.html
java.lang.NoSuchFieldException: index
    at java.lang.Class.getField(Class.java:1549)
    at org.chromium.android_webview.AndroidProtocolHandler.getFieldId(AndroidProtocolHandler.java:67)
    at org.chromium.android_webview.AndroidProtocolHandler.openResource(AndroidProtocolHandler.java:106)
    at org.chromium.android_webview.AndroidProtocolHandler.open(AndroidProtocolHandler.java:52)
```

### Device info

- Device: Teclast tablet
- OS Version: Android 7
- App Manager Version: 3.1.1
- Mode: Root/ADB/NonRoot: root

**Comment by MuntashirAkon** [comment]
I don't think this is fixable on our side. It's either an issue with the WebView or with the OS.

**Comment by 2011** [comment]
My apologies for this. I finally figured out what happened. A while back, I had to reinstall the factory ROM, which contains a very old version of WebView (52). Previously, I had replaced the .apk in the System partition to allow "updates." I had forgotten to do this after the reinstall, and the old WebView simply no longer works. I have no installed an overlay to enable an alternate WebView, which now displays the user manual (and works properly within other apps, as well).
